proposals/0409-access-level-on-imports.md

```diff
@@ -56,7 +56,7 @@ accept access-level modifiers on import declarations to declare the visibility o
 apply that information when type-checking the source file,
 and determine when indirect clients can skip loading transitive dependencies.
 We then cover other concerns addressed by this proposal:
-the different default access levels of imports in Swift 5 and Swift 6,
+the different default access levels of imports in Swift 5 and 6, and a future language mode,
```

Comment:
@Jumhyn Please let me know if you have a suggestion to improve the wording of `Swift 5 and 6, and a future language mode`. I think referencing only Swift 5 would be confusing but this adds some complexity.

Reply:
How about "Swift 5/6 and a future language mode"; otherwise, the first "and" can still be parsed as where the difference lies (i.e., between 5 and 6).

Reply:
Good idea, it makes it much more readable. I've applied your suggestion.